caustic/sources/sersic.py

```diff
@@ -92,9 +94,7 @@ def brightness(self, x, y, params: Optional["Packed"] = None):
             If `lenstronomy_k_mode` is True, we use the approximation from Lenstronomy, 
             otherwise, we use the approximation from Ciotti & Bertin (1999).
         """
-        x0, y0, q, phi, n, Re, Ie = self.unpack(params)
-
-        x, y = translate_rotate(x, y, x0, y0, phi)
+        x, y = translate_rotate(x, y, x0, y0, phi - north)
```

Comment:
Is this the best approach? I think we should do this internally , in the function `translate_rotate`. Otherwise we have to remember to subtract the north everytime we use tranlsate_rotate in future modules.

Reply:
I think this is the more general way to do it. That way calling `translate_rotate(x, y, x0, x0)` will just do translation with no rotation. If we have the `- NORTH` in the function then the default behaviour is to rotate by 90 deg which may be confusing. Also, are there any situations where we want a rotation, but don't need/care about orientation relative to north?

Reply:
Never mind, I can make it work and then yes it will be a lot cleaner.